client/examples/read_to_csv.py

```diff
@@ -17,44 +22,80 @@
             help='Maximum number of samples to record. Default 0 (no maximum)')
     parser.add_argument('-t','--timeout', type=int, default=0,  
             help='Collection time for sampling (s). Default is 0 (no timeout). The nominal sample rate is 5Hz.')
-    parser.add_argument('-p', '--port', default='/dev/ttyACM0',  
-            help='Serial port name. Default is /dev/ttyACM0.')
-    parser.add_argument('filename', help='CSV file for data output')
+    parser.add_argument('-p', '--ports', default='/dev/ttyACM0',  nargs='+',
+            help='Serial port name(s). Default is /dev/ttyACM0.')
+    # parser.add_argument('-f', '--filename', default='test.csv', help='CSV file for data output')
```

Comment:
Would be preferred to keep a base filename, which could be used to provide a prefix i.e. 'test.csv' becomes 'test_COM1_20240101_1234.csv'